--- /etc/my.cnf 2018-09-03 21:49:04.948771044 +0000
+++ my.cnf      2018-09-03 21:45:51.540943371 +0000
@@ -3,6 +3,10 @@ datadir=/var/lib/mysql
 socket=/var/lib/mysql/mysql.sock
 # Disabling symbolic-links is recommended to prevent assorted security risks
 symbolic-links=0
+character-set-server = utf8
+
+[client]
+default-character-set=utf8
 # Settings user and group are ignored when systemd is used.
 # If you need to run mysqld under a different user or group,
 # customize your systemd unit file for mysqld according to the
